source/jquery.flot.js

```diff
@@ -2508,30 +2533,35 @@ Licensed under the MIT license.
                     var found = findNearbyPoint(s, mouseX, mouseY, radius, smallestDistance, computeDistance);
                     if (found) {
                         smallestDistance = found.distance;
-                        item = [i, found.dataIndex];
+                        items.push([i, found.dataIndex]);
                     }
                 }
 
-                if (s.bars.show && !item) { // no other point can be nearby
+                if (s.bars.show && !items.length) { // no other point can be nearby
                     var foundIndex = findNearbyBar(s, mouseX, mouseY);
-                    if (foundIndex >= 0) item = [i, foundIndex];
+                    if (foundIndex) {
```

Comment:
I don't think we usually rely on the truthiness of numeric values in conditionals like this. Can we change this back to `foundIndex >= 0`? Same thing on line 2540.

Reply:
Fixed.